### PR TITLE
East Sand Hole - Right item with bomb escape

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

File: region/maridia/inner-pink/East Sand Hole.json [hunk 1]
```diff
@@ -155,9 +146,29 @@
               "canSunkenTileWideWallClimb"
             ]}
           ]},
+          "canSpringBallJumpMidAir",
+          {"and": [
+            "h_canCrouchJumpDownGrab",
+            "canBombJumpWaterEscape",
+            "h_canIBJ"
```
Comment (osse101) [inline region/maridia/inner-pink/East Sand Hole.json:153]:
```suggestion
            "h_canJumpIntoIBJ"
```

File: region/maridia/inner-pink/East Sand Hole.json [hunk 2]
```diff
@@ -155,9 +146,29 @@
...
+        "canPlayInSand",
+        {"or": [
           {"and": [
-            "HiJump",
-            "canSpringBallJumpMidAir"
+            "h_canMaxHeightSpringBallJump",
+            {"or": [
+              "canWalljump",
+              "can3HighMidAirMorph",
+              "h_canJumpIntoIBJ"
```
Comment (osse101) [inline region/maridia/inner-pink/East Sand Hole.json:170]:
```suggestion
              {"and": [
                "canBombJumpWaterEscape",
                "h_canJumpIntoIBJ"
              ]}
```

File: region/maridia/inner-pink/East Sand Hole.json (hunk 1 above)
Comment (kjbranch) [reply region/maridia/inner-pink/East Sand Hole.json:153]: Is this possible to do with a single jump? I can't get it

File: region/maridia/inner-pink/East Sand Hole.json (hunk 2 above)
Comment (kjbranch) [reply region/maridia/inner-pink/East Sand Hole.json:170]: This is done with spring ball, so it's a lot easier than what I think `canBombJumpWaterEscape` calls for. You can hold jump constantly and just place a well-timed bomb at the peak of any of the jumps

File: region/maridia/inner-pink/East Sand Hole.json (hunk 1 above)
Comment (osse101) [reply region/maridia/inner-pink/East Sand Hole.json:153]: This is a funny way to look at it, but I agree with your reasoning.

With HiJump it looks like your jumping out of the water, and without HJB you are bomb jumping at the water line so they look like they should be switched.

**Pull request merged**